frontend/src/slices/trucks/trucks.slice.ts

```diff
@@ -31,6 +37,35 @@ const { reducer, actions, name } = createSlice({
       })
       .addCase(addTruck.rejected, (state) => {
         state.dataStatus = DataStatus.REJECTED;
+      })
+      .addCase(getTrucksByBusinessId.pending, (state) => {
```

Comment:
It might be better to use .addMatcher instead of .addCase for the pending status. This will make the code a bit more compact. For example:  
```
.addMatcher(
 isAnyOf(addTrucksByUserId.pending, getTrucksByBusinessId.pending, addTruck.pending),
        (state) => {
          state.dataStatus = DataStatus.PENDING;
        },
      )
```